### PR TITLE
Created tests

### PR DESCRIPTION
**Pull request by AyoMoses1**
We created tests for our various components. Please feel free to review and drop your feedbacks where necessary

### CODE REVIEW

File: src/__tests__/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    expect(typeof calculate({}, 'AC')).toBe('object');
+  });
+
+  // it('should return an object with total, next and operation properties', () => {
+  //   expect(calculate({}, 'AC')).toEqual({
+  //     total: null,
+  //     next: null,
+  //     operation: null,
+  //   });
+  // });
```
Comment (VICTOR-CHUKWU) [inline src/__tests__/calculate.test.js]: - [ ] [Optional] kindly remove comments as code

File: src/__tests__/calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    const expected = { total: '-1', next: null, operation: null };
+
+    expect(calculate(obj, '=')).toEqual(expected);
+  });
+
+  it('should multiply two numbers', () => {
+    const obj = { total: '1', next: '-2', operation: 'x' };
+    const expected = { total: '-2', next: null, operation: null };
+
+    expect(calculate(obj, '=')).toEqual(expected);
```
Comment (VICTOR-CHUKWU) [inline src/__tests__/calculate.test.js]: - [ ] Nice approach :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @AyoMoses1 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] all test passed
- [ ] tested relevant component and functions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/calculate.test.js (hunk 2 above)
Comment (AyoMoses1) [reply src/__tests__/calculate.test.js]: Thanks for the feedback. I really appreciate

**Pull request merged**